src/cassette/CassettePort.hh

```diff
@@ -58,6 +58,11 @@ public:
 	*/
 	virtual void setLaserdiscPlayer(LaserdiscPlayer *laserdisc) = 0;
 #endif
+
+	/**
+	* Get the cassette player connected to the port.
+	*/
```

Comment:
I agree with the implementation, but not with the comment.
This method returns the cassette player (if it exists). Whether or not it is plugged into the port.

Reply:
Fixed.